comprehensive_microservice/api_service.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import FastAPI
+
+from .coder import DataCoder, DataModel, Storage
+
+app = FastAPI()
+data_coder = DataCoder(Storage())
```

Comment:
not sure how we would mock it in tests for now, but let's keep it

Reply:
We'll take a look at it later:
https://fastapi.tiangolo.com/advanced/testing-dependencies/